package.json

```diff
@@ -1,12 +1,23 @@
 {
-  "name": "goodparts",
+  "name": "eslint-config-dwyl",
```

Comment:
do we want to double check name with everyone? brings us back to #192 , the name of this module will be a good indication